### PR TITLE
Fix compatibility with schema validation

### PR DESCRIPTION
**Pull request by hstct**
[noissue]

### CODE REVIEW

File: .ci/scripts/validate_commit_message.py [hunk 1]
```diff
@@ -6,14 +7,34 @@
...
+    ".removal",
+    ".misc",
+    ".deprecation",
+    ".translation",
+    ".devel",
+]
 
 sha = sys.argv[1]
-project = "pulp-cli-deb"
+project = "pulp-cli"
```
Comment (mdellweg) [inline .ci/scripts/validate_commit_message.py:31]: This sounds wrong. But is it even used anywhere?

Comment (hstct) [reply .ci/scripts/validate_commit_message.py:31]: Yea I noticed that as well. didn't see it in the diff before the commit. I don't think that it's used somewhere I just copied it from `pulp-cli`. We can remove it and try it out.

Comment (quba42) [reply .ci/scripts/validate_commit_message.py:31]: I would change it back to "pulp-cli-deb" even if we are not sure what it is used for. Changing it from `pulp-cli-deb` to `pulp-cli` just strikes me as wrong.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -55,12 +55,13 @@ jobs:
           - python: "3.7"
             image_tag: "3.20"
           - python: "3.8"
+            pulp_cli: "0.16.0"
             image_tag: "3.19"
           - python: "3.9"
             image_tag: "3.18"
```
Comment (quba42) [inline .github/workflows/main.yml:61]: Remind me again what `pulp_cli` defaults to if we do not explicitly set it?

File: tests/scripts/pulp_deb/test_deb_remote.sh [hunk 3]
```diff
@@ -13,10 +13,10 @@ trap cleanup EXIT
...
 expect_fail pulp deb remote create --name "foo" --url "foo" --distribution ""
 expect_fail pulp deb remote create --name "foo" --url "foo"
-expect_fail pulp deb remote create --name "foo" --url ""    --distribution "foo"
-expect_fail pulp deb remote create --name "foo"             --distribution "foo"
-expect_fail pulp deb remote create --name ""    --url "foo" --distribution "foo"
-expect_fail pulp deb remote create              --url "foo" --distribution "foo"
+expect_fail pulp deb remote create --name "foo" --url "" --distribution "foo"
+expect_fail pulp deb remote create --name "foo" --distribution "foo"
+expect_fail pulp deb remote create --name "" --url "foo" --distribution "foo"
+expect_fail pulp deb remote create --url "foo" --distribution "foo"
```
Comment (quba42) [inline tests/scripts/pulp_deb/test_deb_remote.sh:19]: I don't have a strong opinion, but I would like a consensus going forward if we want to format these test scripts in a manual block aligned style, or if we want to go ahead with these formatting changes?

**Review by quba42 — CHANGES_REQUESTED** [review]
Once we have agreed on the formatting of the test scripts, (and the tests are green) this is good to go.

File: .ci/scripts/validate_commit_message.py (hunk 1 above)
Comment (mdellweg) [reply .ci/scripts/validate_commit_message.py:31]: I'd opt for removing it.

File: .github/workflows/main.yml (hunk 2 above)
Comment (mdellweg) [reply .github/workflows/main.yml:61]: "from source" (the current main branch)

File: .ci/scripts/validate_commit_message.py (hunk 1 above)
Comment (quba42) [reply .ci/scripts/validate_commit_message.py:31]: Removing it is also fine.

File: .github/workflows/main.yml [hunk 4]
```diff
@@ -55,12 +55,13 @@ jobs:
...
           - python: "3.10"
             pulp_cli: "0.15.0"
-            image_tag: "3.17"
+            image_tag: "3.22"
```
Comment (quba42) [inline .github/workflows/main.yml:64]: Ok, I am tired of waiting on this without having a working state in `main` branch! pulp-cli itself does not yet test against 3.22, we do not need to be ahead of pulp-cli!

Let's separate this out into a separate PR, and get everything else merged today!

File: .ci/scripts/validate_commit_message.py (hunk 1 above)
Comment (quba42) [reply .ci/scripts/validate_commit_message.py]: Removed, resolving!

File: .github/workflows/main.yml (hunk 4 above)
Comment (mdellweg) [reply .github/workflows/main.yml]: Yep sorry for the delay. I agree waiting is not a good option at this time.

**Review by quba42 — APPROVED** [review]
